tests/test_parser.py

```diff
@@ -0,0 +1,208 @@
+import pytest
+from decimal import Decimal\
+
+FILE_LOCATION = './transactions-0043733f.xdr.gz'
+
+
+@pytest.fixture('session')
+def xdr_data():
+    with open(FILE_LOCATION, 'rb') as f:
+        data = f.read()
+    return data
+
+
+def test_init_unpacker(xdr_data):
+    from xdrparser.parser import init_unpacker
+    unpacker, methods = init_unpacker(xdr_data)
+    assert len(unpacker._Unpacker__buf) == 414129
+    for xdr_type in ['bucket', 'ledger', 'transactions', 'results', 'scp']:
+        assert xdr_type in methods
+
+
+def test_unpack_file():
+    from xdrparser.parser import unpack_file
+    unpacked = unpack_file(FILE_LOCATION)
+    assert len(unpacked) == 64
+
+
+def test_parse():
+    from xdrparser.parser import parse
+    parse(FILE_LOCATION)
```

Comment:
I think you should save the expected output in some file and assert that you get the same thing